### PR TITLE
react-native-push-notification: add picture property to notification object type

### PR DESCRIPTION
**Pull request by whenmoon**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

Adding picture type to local notification object interface: https://github.com/zo0r/react-native-push-notification#local-notifications

### CODE REVIEW

File: types/react-native-push-notification/index.d.ts [hunk 1]
```diff
@@ -84,6 +84,7 @@ export class PushNotificationObject {
     id?: string | number | undefined;
     title?: string | undefined;
     message: string;
+	picture?: string;
```
Comment (DangerBotOSS) [inline types/react-native-push-notification/index.d.ts:87]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/react-native-push-notification/index.d.ts [hunk 2]
```diff
@@ -84,6 +84,7 @@ export class PushNotificationObject {
     id?: string | number | undefined;
     title?: string | undefined;
     message: string;
+    picture?: string;
```
Comment (jakebailey) [inline types/react-native-push-notification/index.d.ts:87]: Judging by the other properties in this object, I'm guessing this needs to be:
```suggestion
    picture?: string | undefined;
```

**Review by jakebailey — CHANGES_REQUESTED** [review]

File: types/react-native-push-notification/index.d.ts (hunk 2 above)
Comment (whenmoon) [reply types/react-native-push-notification/index.d.ts]: Thanks @jakebailey suggestion committed.

File: types/react-native-push-notification/index.d.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-native-push-notification 7.3
+// Type definitions for react-native-push-notification 8.1
```
Comment (jakebailey) [inline types/react-native-push-notification/index.d.ts:1]: Just noticed this line; you're bumping this up to another major version. Was that intentional?

If so, I believe there's a little more you have to do in order to keep v7 in the repo.

Comment (jakebailey) [reply types/react-native-push-notification/index.d.ts:1]: > If so, I believe there's a little more you have to do in order to keep v7 in the repo.

This turned out to be wrong; they only have to be copied once someone wants to change v7 again.

In any case, let me know if this was intentional.

**Pull request merged**